morse_code.rb

```diff
@@ -0,0 +1,31 @@
+def decode_char(str)
+  collection = {
+    'A' => '.-', 'B' => '-...', 'C' => '-.-.', 'D' => '-..', 'E' => '.', 'F' => '..-.',
+    'G' => '--.', 'H' => '....', 'I' => '..', 'J' => '.---', 'K' => '-.-', 'L' => '.-..',
+    'M' => '--', 'N' => '-.', 'O' => '---', 'P' => '.--.', 'Q' => '--.-', 'R' => '.-.',
+    'S' => '...', 'T' => '-', 'U' => '..-', 'V' => '...-', 'W' => '.--', 'X' => '-..-',
+    'Y' => '-.--', 'Z' => '--..'
+  }
+
+  collection.key(str) || ''
+end
+
+def decode_word(str)
+  new_str = ''
+  str.split.each do |char|
+    new_str += decode_char(char)
+  end
+  new_str
+end
+
+def decode_result(str)
+  result = ''
+  str.split('  ').each do |word|
+    result += "#{decode_word(word)} "
+  end
+  result
+end
+
+puts decode_char('.-')
+puts decode_word('-- -.--')
+puts decode_result('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```

Comment:
- [ ] You printing fewer lines than what we are expecting you to print. You were supposed to print the following patter:
![Screenshot from 2022-07-27 20-44-45](https://user-images.githubusercontent.com/56886064/181348760-24fa8e7b-1992-46f4-899b-9494d25d30ff.png)

But you printed this one:
![Screenshot from 2022-07-27 20-43-55](https://user-images.githubusercontent.com/56886064/181348850-b91fa57b-4a7a-450c-9dbf-e6a8cdf92073.png)

Would you fix it before you submit for another review?